Project.toml

```diff
@@ -34,6 +34,7 @@ Geodesy = "1"
 GeometryBasics = "0.1 - 0.4"
 Glob = "1.2 - 1.3"
 ImageIO = "0.1 - 0.6"
+JLD2 = "0.4"
```

Comment:
Is it not possible to use an older JLD2 version as well? Say 0.2? if you make this too recent it may needlessly cause collisions with other packages?

Reply:
0.4 was released in 2021, so I thought this should be enough. But I can also use an older version.